config/settings.py

```diff
@@ -632,3 +650,37 @@
     raise ImproperlyConfigured(
         "SITE_BANNER_STYLE accepts only one of 'local', 'testing', 'production'."
     )
+
+# Feature Flags
+# -----------------------------------------------------------------------------
+# These flags are used to enable/disable features in AMY.
+# See https://cfpb.github.io/django-flags/ for more details.
+
+# ------------
+# The system for enabling or disabling feature flags by users themselves should only be
+# used if the feature flag can be enabled by a single parameter condition set to `=true`
+# Disabling the feature flag should be done by setting the URL parameter to `=false`.
+# There should also be included a session condition. For example:
+#
+#     FLAGS = {
+#         "EMAIL_MODULE": [
+#             {"condition": "anonymous", "value": False, "required": True},
+#             {"condition": "parameter", "value": "enable_email_module=true"},
+#             {"condition": "session", "value": "enable_email_module"},
+#         ],
+#     }
+# ------------
+FLAGS = {
+    # Enables instructor recruitment views; this should be enabled in all environments.
+    # The flag itself was not migrated from envvar to django-flags mechanism.
+    # "INSTRUCTOR_RECRUITMENT": [{"condition": "boolean", "value": True}],
+    # ------------
+    # Enables the new email module. It's only for logged-in users and it has to be
+    # enabled through `?enable_email_module` GET parameter or the same parameter stored
+    # in request session.
+    "EMAIL_MODULE": [
```

Comment:
How would we enable this flag globally in production? 

I looked at the [django-flags usage guide](https://cfpb.github.io/django-flags/usage/) and saw that "Additional conditions can be added in the Django admin for any defined flag." I think that would be a good approach, so I tried setting a boolean True condition in the Django admin on my local instance, but it didn't seem to have any effect.

Reply:
It works for me with this setting:
<img width="522" alt="image" src="https://github.com/carpentries/amy/assets/72821/d15078ba-32da-4db2-8570-d18beed00c4b">

I'm even unable to disable the flag using my own feature flags page 😄

Reply:
Ah, so it does. I had set "Required" to be true, and that prevented the flag from being enabled. I'm not sure why.

Reply:
Honestly I don't know that either. I think all conditions marked as required must be true, or only one non-required must be true for the flag to be enabled. https://cfpb.github.io/django-flags/usage/#defining-flags